Utilities/CodeGeneration/EdFi.Ods.CodeGen/Database/DatabaseSchema/AuthorizationDatabaseTableViewsProvider.cs

```diff
@@ -0,0 +1,87 @@
+﻿// SPDX-License-Identifier: Apache-2.0
+// Licensed to the Ed-Fi Alliance under one or more agreements.
+// The Ed-Fi Alliance licenses this file to you under the Apache License, Version 2.0.
+// See the LICENSE and NOTICES files in the project root for more information.
+
+using System.Collections.Generic;
+using System.Linq;
+using DatabaseSchemaReader;
+using DatabaseSchemaReader.DataSchema;
+using EdFi.Common;
+using EdFi.Common.Database;
+using EdFi.Ods.CodeGen.Models;
+using EdFi.Ods.CodeGen.Providers;
+
+namespace EdFi.Ods.CodeGen.Database.DatabaseSchema
+{
+    public class AuthorizationDatabaseTableViewsProvider : IAuthorizationDatabaseTableViewsProvider
+    {
+        private readonly string _connectionString;
+        private readonly IDatabaseConnectionProvider _databaseConnectionProvider;
+        private List<AuthorizationDatabaseTable> _views;
+
+        public AuthorizationDatabaseTableViewsProvider(
+            IDatabaseConnectionProvider databaseConnectionProvider,
+            IDatabaseConnectionStringProvider connectionStringProvider)
+        {
+            _databaseConnectionProvider = Preconditions.ThrowIfNull(
+                databaseConnectionProvider, nameof(databaseConnectionProvider));
+
+            Preconditions.ThrowIfNull(connectionStringProvider, nameof(connectionStringProvider));
+            _connectionString = connectionStringProvider.GetConnectionString();
+        }
+
+        public List<AuthorizationDatabaseTable> LoadViews()
+        {
+            if (_views != null)
+            {
+                return _views;
+            }
+
+            using (var connection = _databaseConnectionProvider.CreateConnection(_connectionString))
+            {
+                connection.Open();
+
+                using (var dr = new DatabaseReader(connection))
+                {
+                    dr.CommandTimeout = 60;
+                    var dataTupleTable = dr.Table("EducationOrganizationIdToEducationOrganizationId");
```

Comment:
Preferably move this line down to just above where you're adding it to the views list.  Its generally preferable to keep variable setup as close as you can to where you're using it, rather than initializing it up here, setting up all the view data, then coming back to use the table data.

Reply:
I fixed it .Thanks